README.md

```diff
@@ -78,26 +78,39 @@ This module exports a single function.
 
 ```js
 const plugin = createPlugin(
-  globals: Object,
+  globals: { [id: string]: string } | (id: string) => string,
```

Comment:
1. Since the key of an object is always a string, there is no need to reference its type.
2. The function is allowed to return undefined (`void`).

Actually, I think the signature of the function is pretty clear below. How about to simply write `Object | Function`?